Detectors/TPC/reconstruction/macro/RawClusterFinder.C

```diff
@@ -0,0 +1,550 @@
+#ifndef ALICEO2_TPC_CONVERTRAWCLUSTERS_C_
```

Comment:
why is this in a macro? It looks to me more like library code (in contrast to configuration).